backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
```

Comment:
Gonna take a stab and rewriting/combining these first two/three paragraphs: 

```suggestion
The {oadp-first} product safeguards customer applications on {product-title}. It offers comprehensive disaster recovery protection, covering OpenShift Container Platform applications, application-related cluster resources, persistent volumes, and internal images. OADP is also capable of backing up both containerized applications and virtual machines (VMs). However, it does not serve as a disaster recovery solution for etcd or OpenShift Operators. 
```

Reply:
Thanks for the re-work

I would like to keep the mention of etc highlighted, using whitespace. So I am going to drop that line down.